src/net.cpp

```diff
@@ -930,21 +930,6 @@ size_t CConnman::SocketSendData(CNode& node)
     return nSentSize;
 }
 
-struct NodeEvictionCandidate
```

Comment:
20477: missing added optional include? probably compiles fine due to other headers already including; but we should IWYU